db/data.py

```diff
@@ -460,3 +461,32 @@ def get_summary_data(mbid, offset=0):
         pass
 
     return summary
+
+def check_low_level_duplicates(data):
+    """
+        Check if data comes from an audio file already analysed
+        Uses the md5_encoded field
+    Args: data
+        A dictionary containing the low-level data extracted using essentia
+    """
+    if 'md5_encoded' in data['metadata']['audio_properties']:
+        input_md5 = data['metadata']['audio_properties']['md5_encoded']
+        query = text("select data->'metadata'->'tags'->'musicbrainz_recordingid' from lowlevel_json where data->'metadata'->'audio_properties'->'md5_encoded' ? :md5 ")
```

Comment:
Look at the way that we format SQL statements in this file, especially keywords in capitals, and the position of keywords, fields, and commas:
https://github.com/hellska/acousticbrainz-server/blob/e5ad20e9d4807abbe4be030c104a2a1531bcd9ab/db/data.py#L135-L138